### DISCUSSION TITLE
Problem using `SFTPGO_COMMAND__COMMANDS...` environment variables

### DISCUSSION THREAD

**Discussion by DrMurx** [drakkan/sftpgo#1337] [Q&A] [answered]
I've configured these environment variables:

```
SFTPGO_COMMON__ACTIONS__EXECUTE_ON: upload
SFTPGO_COMMON__ACTIONS__HOOK: /scripts/my_script.sh
SFTPGO_COMMAND__COMMANDS__0__PATH: /scripts/my_script.sh
SFTPGO_COMMAND__COMMANDS__0__ARGS: my_argument
SFTPGO_COMMAND__COMMANDS__0__ENV: FOO=bar
```

The `my_script.sh` command is correctly executed after an upload, and it correctly receives the value `bar` in an environment variable `$FOO`, but it's not launched with the command line argument `my_argument`.

This is the corresponding snippet from the debug output
```
config file used: '\"/etc/sftpgo/sftpgo.json\"', config loaded:
...
Commands:[{Path:/scripts/my_script.sh Timeout:0 Env:[FOO=bar] Args:[] Hook:}]}
```

The config loader correctly parsed the `SFTPGO_COMMAND__COMMANDS__0__ENV` value into a single element array, but it completely ignores the `SFTPGO_COMMAND__COMMANDS__0__ARGS`.

**Comment by drakkan** [comment, marked as answer]
https://github.com/drakkan/sftpgo/commit/d1e4ee7bc824d2fb7f870da04df282af0ad3fa32

**Comment by DrMurx** [comment]
That was fast, thanks!
